bugsnag-plugin-android-ndk/src/main/assets/include/bugsnag.h

```diff
@@ -8,6 +8,9 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+
+typedef bool (*on_error)(intptr_t);
```

Comment:
Do all our types have `bsg_` prefix - maybe `bsb_on_error_cb_t`?

Reply:
Renamed the type as `bsg_on_error` - this is consistent with how we name other structs/types.